stable/app/templates/migration-job.yaml

```diff
@@ -0,0 +1,49 @@
+{{- if .Values.migration.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "app.fullname" . }}-migration-job
+  labels:
+    {{- include "app.labels" . | nindent 4 }}
+spec:
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "app.selectorLabels" . | nindent 8 }}
+    spec:
+      restartPolicy: Never
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
+          command:
+            {{- toYaml .Values.migration.command | nindent 12 }}
+          args:
+            {{- toYaml .Values.migration.args | nindent 12 }}
```

Comment:
In the deployment template we are using `entryPoint` instead of `args`.
I think we should refactor all to use same even if it requires a breaking change.

Reply:
agreed, should we keep the changes to follow kubernetes fields or back to `entrypoint`? IMHO entrypoint could be misleading as the `args`.

Also it seems like only `guardian` chart is using this base chart.
cc: @rahmatrhd

Reply:
Let's go with `args`. Since `guardian` is version locking the dependency, we can make this breaking change without affecting it.
We will also need to make a breaking change on `guardian` chart as well.